test/train-sets/ref/help.stdout

```diff
@@ -406,6 +406,9 @@ importance weight classes:
 Confidence:
   --confidence                 Get confidence for binary predictions
   --confidence_after_training  Confidence after training
+Generate interactions:
```

Comment:
should this say `Generate permutations` or `Generate permutated interactions`?

Reply:
Interactions are either permutations or combinations depending on the arguments. This should probably actually say "Count interacted features"